article/_posts/2019-05-30-Speech2Face.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: review
+title: "Speech2Face: Learning the Face Behind a Voice"
+tags: deep-learning
+author: "Antoine Théberge"
+cite:
+    authors: "Tae-Hyun Oh, Tali Dekel, Changil Kim, Inbar Mosseri, William T. Freeman, Michael Rubinstein, Wojciech Matusik"
+    title:   "Speech2Face: Learning the Face Behind a Voice"
+    venue:   "CVPR 2019"
+pdf: "https://arxiv.org/pdf/1905.09773v1.pdf"
+---
+
+
+# Highlights
+
+- Successfully recognize general physical traits such as gender, age, and ethnicity from a voice clip
+
+# Introduction
+
+"_How much can we infer about a person's looks from the way they speak?_". In this paper, the authors reconstruct a "canonical" (front facing, neutral expression, uniformly lit) face from a 6 seconds voice clip using a voice encoder.
+
+# Methods
+
+![](/article/images/Speech2Face/method.jpeg)
+
+The idea is really simple: You take a pre-trained face synthetiser<sup>[1]</sup> network. You then train a voice encoder to match its last feature vector $$v_s$$ with the face synthesiser $$v_f$$. If the two encoders project in a similar space, the face decoder should decode similar faces.
+
+A natural choice for a loss would be the $$L_1$$ distance between the $$v_s$$ and $$v_f$$. However, the authors found that the training was slow and unstable. They, therefore, used the following loss:
+
+![](/article/images/Speech2Face/loss.jpeg)
+
+where $$f_{dec}$$ is the first layer of the face decoder and $$f_{VGG}$$ the last layer of the face encoder.
+
+## Data
+
+The authors used the AVSpeech dataset and extracted 1 frame and a 6 seconds audio clip from each video clip. If the video clip was shorter than six seconds, they looped it. The audio clip was then turned into a spectrogram and fed to the face encoder.
```

Comment:
The spectrogram is fed into the voice encoder, right?

Reply:
Woops you're right